packages/mg-ghost-api/lib/processor.js

```diff
@@ -0,0 +1,61 @@
+module.exports.processTerms = (ghTags) => {
+    let tags = [];
+
+    ghTags.forEach((tag) => {
+        tags.push({url: tag.url, data: tag});
+    });
+
+    tags.push({
+        url: 'migrator-added-tag', data: {name: '#ghost'}
+    });
+
+    return tags;
+};
+
+module.exports.processPost = (ghPost) => {
+    const post = {
+        url: ghPost.url,
+        data: ghPost
+    };
+
+    post.data.authors = this.processAuthors(ghPost.authors);
+
+    if (ghPost.tags && ghPost.tags.length > 0) {
+        post.data.tags = this.processTerms(ghPost.tags);
+    } else {
+        delete post.data.tags;
+    }
+
+    return post;
+};
+
+module.exports.processPosts = (posts) => {
+    return posts.map(post => this.processPost(post));
+};
+
+module.exports.processAuthor = (ghAuthor) => {
+    let profileImage = ghAuthor.profile_image.replace(/s=96/, 's=3000');
```

Comment:
This threw an error for me on a test run:

```
  TypeError: Cannot read property 'replace' of null
      at Object.module.exports.processAuthor (/Users/aileen/code/migrate/packages/mg-ghost-api/lib/processor.js:37:47)
      at /Users/aileen/code/migrate/packages/mg-ghost-api/lib/processor.js:51:39
      at Array.map (<anonymous>)
      at Object.module.exports.processAuthors (/Users/aileen/code/migrate/packages/mg-ghost-api/lib/processor.js:51:20)
      at Object.module.exports.all (/Users/aileen/code/migrate/packages/mg-ghost-api/lib/processor.js:57:25)
      at Task.task (/Users/aileen/code/migrate/packages/migrate/sources/ghost.js:81:57)
      at /Users/aileen/code/migrate/node_modules/listr/lib/task.js:167:30 {
    context: {
      errors: [Circular],
      options: [Object],
      fileCache: [FileCache],
      imageScraper: [ImageScraper],
      linkFixer: [LinkFixer],
      result: [Object]
    }
  }
```

Ran the test for mainframe: 
`yarn migrate ghost https://mainframe.ghost.io <apiKey>  --postFilter 'tag:[hash-faq]' --pages false --verbose`